### PR TITLE
add responsive reviews slider

### PR DESCRIPTION
**Pull request by Saminsfw**
review-desktop
review-tab
review-mobile

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <p class="title">Our Reviews</p>
+        <hr />
+        <article class="review">
+          <img src="person-1.jpeg" id="person-img" alt="" />
```
Comment (UmmarHamid) [inline index.html]: @Saminsfw What if use an image from api as you did in the reviewsObject and then delete the image altogether?

**Review by UmmarHamid — APPROVED** [review]
LGTM!

File: index.html (hunk 1 above)
Comment (Saminsfw) [reply index.html]: Done

Comment (UmmarHamid) [reply index.html]: I can still see the image, can you please remove that?

**Review by UmmarHamid — APPROVED** [review]

**Pull request merged**